demos/covid-19/covid_19_seird/model/sys_params.py

```diff
@@ -0,0 +1,12 @@
+sys_params = {
```

Comment:
It will be nice if you include the meaning of D (number of days an infected person has and can spread the disease?).